high_school/templates/high_school/index.html

```diff
@@ -151,14 +149,35 @@
         {#to toggle fav high school for user#}
         $(".school-fav-btn").click(function (e) {
             e.preventDefault();
+            let school_fav_btn = $(this);
             let school_dbn = $(this).attr("dbn_value");
             let is_fav = parseInt($(this).attr("is_fav"));
             is_fav = is_fav === 1 ? 0 : 1;
-            $(this).attr("is_fav", is_fav);
+            school_fav_btn.attr("is_fav", is_fav);
             let url = "{% url 'dashboard:high_school:toggle_fav' 'school_dbn' '0000' %}";
             url = url.replace('school_dbn', school_dbn);
             url = url.replace('0000', "" + is_fav);
-            window.location.href = url;
+            let is_fav_on = "{{ request.GET.is_fav_on }}";
+            $.ajax({
+                url: url,
+                type: "POST",
+                success: function (data) {
+                    console.log("success " + data);
+                    let i_tag = school_fav_btn.find("i");
+                    i_tag.removeClass("fa-heart-o");
+                    i_tag.removeClass("fa-heart");
+                    i_tag.addClass(is_fav ? "fa-heart" : "fa-heart-o");
+                    if (is_fav_on) {
+                        show_toast("The page will reload in 5 seconds", true);
+                        setTimeout(function () {
+                            window.location.reload();
+                        }, 5000);
+                    }
+                },
+                error: function (data) {
+                    console.log("error")
```

Comment:
do not print errors on console.

Reply:
This was added to cover when the user has selected to view his favourites and they toggle one of the school from their fav. It'll toggle on the screen but to ensure consistency we should reload the page, because otherwise it might feel like there's nothing happening on clicking the heart icon on a school item.

Reply:
But users might not look at the console logs.

Reply:
Oh, this is my bad. My phone showed your comment on the toast logic in the lines above instead of this console line. I'll get rid of the logs, sorry.